go/preload/main/preload.go

```diff
@@ -0,0 +1,110 @@
+package main
+
+import (
+	"flag"
+	"log"
+	"regexp"
+	"sync"
+
+	"github.com/google/certificate-transparency/go/client"
+	"github.com/google/certificate-transparency/go/scanner"
+)
+
+var sourceLogUri = flag.String("source_log_uri", "http://ct.googleapis.com/aviator", "CT log base URI to fetch entries from")
+var targetLogUri = flag.String("target_log_uri", "http://example.com/ct", "CT log base URI to add entries to")
+var blockSize = flag.Int("block_size", 1000, "Max number of entries to request at per call to get-entries")
+var numWorkers = flag.Int("num_workers", 2, "Number of concurrent matchers")
+var parallelFetch = flag.Int("parallel_fetch", 2, "Number of concurrent GetEntries fetches")
+var parallelSubmit = flag.Int("parallel_submit", 2, "Number of concurrent add-[pre]-chain requests")
+var startIndex = flag.Int64("start_index", 0, "Log index to start scanning at")
+var quiet = flag.Bool("quiet", false, "Don't print out extra logging messages, only matches.")
```

Comment:
Shouldn't that be better controlled by some ```--v=1``` style thing (as in, by the logging library itself)?